client/src/pages/result/Result.js

```diff
@@ -0,0 +1,73 @@
+import {  useEffect, useState } from "react";
+import rawData from "../../data/kujiHelplineSheet.json";
+import Table from "react-bootstrap/Table";
+import  "./Result.css";
+import Form from "react-bootstrap/Form";
+export default function Result() {
+  const [service,setService]=useState("Legal Aid");
+  const [location,setLocation]=useState("North");
+  const [data, setData] = useState([]);
+    useEffect(()=>{
+        function getData(){
+         setData(rawData);
+        }
+    getData();
+    },[]);
+
+    function selectService(e) {
+      setService(e.target.value);
+    }
+
+    function selectLocation(e) {
+      setLocation(e.target.value);
+    }
+  return (
+    <>
+    <Form>
+      <Form.Group className="select-group" controlId="service">
+        <Form.Label>Service</Form.Label>
+        <Form.Select aria-label="Service" value={service} onChange={selectService}>
+          {[...new Set(data.map((item) => item.Themes))].map((item,index)=>(
+            <option  key={index}>{item}</option>
```

Comment:
Very nice use of `Set`!
The `key` should be something that uniquely identifies the object. While the index is sometimes a viable option if there's no other in this case we can use the `item` itself.